subscription_test.go

```diff
@@ -20,6 +20,9 @@ import (
 	"testing"
 	"time"
 
+	"net"
```

Comment:
There should be some better way how to check format automatically (`make format` does not help in this case) at least using goimports (they at least sort imports) - as this imports fiddling is very error prone